### DISCUSSION TITLE
[Feature Request] add support for VFDM Displays

### DISCUSSION THREAD

**Discussion by thomas374b** [arendst/Tasmota#21761] [Support request]
Hello,

in the recent past I created an additional display driver for a VFDM Display device in my fork of tasmota.  It is a 20x2 textual device that is operated with one single serial line @19200 Baud.

Is it possible to merge it upstream ?

I created a branch (in my fork of tasmota) where only those changes regarding this display driver are contained. The branch was based on 13.2 Version.

I could prepare a pull request if this is desired.

Best Regards

**Comment by Jason2866** [comment]
Do you have a link to such a Display? My search was without any usefull hits.

**Comment by thomas374b** [comment]
Hello Jason,

Am Thu, 11 Jul 2024 11:18:29 -0700
schrieb Jason2866 ***@***.***>:

> Do you have a link to such a Display? My search was without any
> usefull hits.

https://www.giant-supplier.com/GIANT-SUPPLIER/vfd-graphic1.html

has a lot of these listed

I have this

   20L203DA5

It can be bought here

https://german.alibaba.com/product-detail/20L203DA6C-20L203DA7-20L203DA5-VFM202LDS0-VFD-DISPLAY-1555581175.html

Similar devices are

   20M102DA1


I attached the datasheet.


Best Regards

Thomas Pantzer
